restws_basic_auth/restws_basic_auth.info

```diff
@@ -1,3 +1,5 @@
 name = Basic authentication login
 description = User login from HTTP authorization headers (part of RESTful web services).
+core = 7.x
+files[] = restws_basic_auth.test
```

Comment:
This is not needed in Backdrop. If you put your `MODULE.test.info` file in the folder `tests` everything will be autoloaded. This is new, and different from D7.

Check out: https://docs.backdropcms.org/documentation/testing-framework